specification/search/data-plane/Azure.Search/readme.md

```diff
@@ -171,6 +171,58 @@ directive:
       - RequiredReadOnlyProperties
 ```
 
+### Suppress custom user schema model violations
+Many Search operations return a custom user schema for the documents in an index.  Some of them are incorrectly specified as `"additionalProperties": { "type": "object" }` instead of `"additionalProperties": true`.  We’ve filed https://github.com/Azure/azure-rest-api-specs/issues/23807 to track fixing these, but are temporarily suppressing the model violation warnings to avoid impacting the next release our generated client libraries.
+
+
+``` yaml
+directive:
+  - suppress: INVALID_TYPE
+    from: searchservice.json
+    reason: Temporarily ignore warnings about incorrectly specified custom user schemas.
+  - suppress: INVALID_TYPE
+    from: searchindex.json
+    reason: Temporarily ignore warnings about incorrectly specified custom user schemas.
+```
+
+### Tag: package-2023-07-01-preview
+
+These settings apply only when `--tag=package-2023-07-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2023-07-01-preview'
+input-file:
+- preview/2023-07-01-Preview/searchservice.json
+- preview/2023-07-01-Preview/searchindex.json
+```
+
+#### Rename one of SearchError definitions
+
+SearchError is duplicated between two swaggers, rename one of them
+
+``` yaml $(tag) == 'package-2023-07-01-preview'
+directive:
+- from: preview/2023-07-01-Preview/searchservice.json
+  where: $.definitions.SearchError
+  transform: $["x-ms-client-name"] = "SearchServiceError"
+```
+
+### Tag: package-2023-07-searchservice-preview
+
+These settings apply only when `--tag=package-2023-07-searchservice-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2023-07-searchservice-preview'
+input-file:
+- preview/2023-07-01-Preview/searchservice.json
+```
+
+### Tag: package-2023-07-searchindex-preview
+
+These settings apply only when `--tag=package-2023-07-searchindex-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2023-07-searchindex-preview'
+input-file:
+- preview/2023-07-01-Preview/searchindex.json
+
 ### Tag: package-2021-04-30-preview
```

Comment:
Nit - you need to close this code block.
``````suggestion
``` yaml $(tag) == 'package-2023-07-searchindex-preview'
input-file:
- preview/2023-07-01-Preview/searchindex.json
```

### Tag: package-2021-04-30-preview
``````